extensions/spotify-beta/src/nowPlayingMenuBar.tsx

```diff
@@ -34,8 +35,17 @@ import { useCachedState } from "@raycast/utils";
 import { useCurrentlyPlayingUri } from "./hooks/useCurrentlyPlayingUri";
 import { formatTitle } from "./helpers/formatTitle";
 
+enum NowPlayingMenuBarIconType {
+  Spotify = "spotify",
+  Album = "album",
+}
+
 function NowPlayingMenuBarCommand({ launchType }: LaunchProps) {
-  const preferences = getPreferenceValues<{ maxTextLength?: boolean; showEllipsis?: boolean }>();
+  const preferences = getPreferenceValues<{
+    maxTextLength?: boolean;
+    showEllipsis?: boolean;
+    iconType?: NowPlayingMenuBarIconType;
```

Comment:
Can we use string union instead of enum please? It's what I've been using throughout the codebase. Thanks!

Reply:
```suggestion
    iconType?: "spotify-icon" | "album-cover";
```